types/data.d.ts

```diff
@@ -42,6 +42,7 @@ export interface RaidbossData {
   CanCleanse: () => boolean;
   CanFeint: () => boolean;
   CanAddle: () => boolean;
+  partyMemberParam: (name: string) => PartyMemberParamObject | undefined;
```

Comment:
Just to avoid polluting data even more, what do you think about making this be a function on `PartyTracker` since `data.party` is already a thing? I'd be fine with `data.party.member(name)` if you want to keep it brief.

I know this would require passing in nicknames and the default field as well, so maybe it's not ideal. Mostly curious if you think that's polluting party tracker with things it shouldn't know about or if that's cleaner.

Reply:
I think the only way I'd be okay with this would be to pass the whole `RaidbossOptions` object in, rather than passing in individual values, and AFAIK that would cause issues with oopsy and jobs since their config options don't have that parameter?

Reply:
oopsy has a nicknames field. It could arguably grow the default field and so oopsy could say people's roles rather than names. jobs doesn't really have a need for either. So, maybe it could take an optional object that has nick name and default that both `RaidbossOptions` and `OopsyOptions` is shaped like. (Or an object with optional nick name and default?)

Reply:
My apologies if I derailed this PR with these comments. Mostly I just am trying to be cautious about adding unnecessary things to `data` since it's something shared with users. Happy to discuss more if you have time!

I'm guessing you're just very busy these days; please let me know if there's something I can do to help this along, as I'm still really excited about this change and being able to let users have more custom output for players.

Reply:
Yeah, I've been very busy the last month or so, so I haven't had time to circle back around to this PR and figure out how to get things to pass through well.

I'm hoping to have time in the next week or two, but we'll see how work goes 🤷

Reply:
Addressed this in c77169acd05e99a6a90b660b393542402219f61b